network/topics.go

```diff
@@ -23,12 +23,23 @@ import (
 	"github.com/algorand/go-algorand/crypto"
 )
 
+// Constant strings used as keys for topics
+const (
+	requestHashKey = "RequestHash"
+	ErrorKey       = "ERR" // used for passing an error message
```

Comment:
nit: all the reminder of the topics are nicely formatted string, with the exception for this one. shouldn't we make it into `Error` to align the formatting to be the same ?